sonar-php-plugin/src/main/java/org/sonar/plugins/php/core/NoSonarAndCommentedOutLocSensor.java

```diff
@@ -67,38 +69,37 @@ public NoSonarAndCommentedOutLocSensor(FileSystem filesystem, NoSonarFilter noSo
     this.filter = noSonarFilter;
     this.filesystem = filesystem;
     this.filePredicates = filesystem.predicates();
+  }
 
+  @Override
+  public void describe(SensorDescriptor descriptor) {
+    descriptor.name(NAME);
   }
 
-  /**
-   * @see org.sonar.api.batch.Sensor#analyse(org.sonar.api.resources.Project, org.sonar.api.batch.SensorContext)
-   */
-  public void analyse(Project project, SensorContext context) {
+  @Override
+  public void execute(SensorContext context) {
     Iterable<InputFile> sourceFiles = filesystem.inputFiles(filePredicates.and(filePredicates.hasLanguage(Php.KEY), filePredicates.hasType(InputFile.Type.MAIN)));
     for (InputFile file : sourceFiles) {
       // TODO: remove when deprecated NoSonarFilter will be replaced.
-      org.sonar.api.resources.File phpFile = context.getResource(org.sonar.api.resources.File.create(file.relativePath()));
+      org.sonar.api.resources.File phpFile = org.sonar.api.resources.File.create(file.relativePath());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "File"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "create"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)